src/test/unit/profile-upgrading.test.js

```diff
@@ -304,3 +304,27 @@ describe('upgrading gecko profiles', function() {
     expect(geckoProfile12).toEqual(afterUpgradeGeckoReference);
   });
 });
+
+describe('importing perf profile', function() {
+  it('should import a perf profile', function() {
+    let version = -1;
+    try {
```

Comment:
Can you explain why this try block is here? Initially I thought it would swallow any errors, but then I saw the version check at the end. It seems like it would be better to just let it fail, as the test would fail, then we would get the error in the test runner output.

Reply:
Could you still address this comment here?